src/publicodes-state/types.d.ts

```diff
@@ -45,6 +45,7 @@ export type Tutorials = Record<string, boolean>
 export type ComputedResultsFootprint = {
   bilan: number
   categories: Record<DottedName, number>
+  subcategories: Record<DottedName, number>
```

Comment:
Maybe you should mark it optional in a first time as not all the simulation computedResults have it ??

Reply:
👍